src/Resources/contao/config/config.php

```diff
@@ -67,6 +67,8 @@
     unset($GLOBALS['BE_MOD']['design']['i18nl10n']);
 }
 
+// ComponentStyleSelect override
+$GLOBALS['BE_FFL']['stylemanager'] = 'WEM\\SmartgearBundle\\Widgets\\ComponentStyleSelect';
```

Comment:
Try to use `WEM\SmartgearBundle\Widgets\ComponentStyleSelect::class` instead

Reply:
done, it works